packages/shared/src/components/feeds/FeedSettings/sections/FeedSettingsTagsSection.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { ReactElement, useContext, useMemo, useState } from 'react';
+import { FeedSettingsEditContext } from '../FeedSettingsEditContext';
+import { TagSelection } from '../../../tags/TagSelection';
+import { Origin } from '../../../../lib/log';
+import { TagElement } from '../TagElement';
+import {
+  ModalKind,
+  ModalPropsContext,
+  ModalSize,
+} from '../../../modals/common/types';
+import { ModalTabs } from '../../../modals/common/ModalTabs';
+import useDebounceFn from '../../../../hooks/useDebounceFn';
+import { useTagSearch } from '../../../../hooks/useTagSearch';
+import { SearchField } from '../../../fields/SearchField';
+import { TagsData } from '../../../../graphql/feedSettings';
+import {
+  Typography,
+  TypographyColor,
+  TypographyType,
+} from '../../../typography/Typography';
+import useFeedSettings from '../../../../hooks/useFeedSettings';
+import useTagAndSource from '../../../../hooks/useTagAndSource';
+
+const tabs = ['Suggested', 'My tags'];
```

Comment:
I think we can put this in an enum to make it safer to work with. Wdyt?

Reply:
yes